<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>419</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>419</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gobang</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/close.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="min_btn">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>0</y>
      <width>25</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/min.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>419</width>
      <height>630</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/cover.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/name.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AiGame_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/aigame.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="LocalGame_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/localgame.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="NetGame_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/img/netgame.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>min_btn</zorder>
   <zorder>close_btn</zorder>
   <zorder>label_2</zorder>
   <zorder>AiGame_btn</zorder>
   <zorder>LocalGame_btn</zorder>
   <zorder>NetGame_btn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>released()</signal>
   <receiver>Menu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_btn</sender>
   <signal>released()</signal>
   <receiver>Menu</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
